Laboratory/Rust/benchmarking/src/bebops/jazz.rs

```diff
@@ -491,3 +491,474 @@ impl<'raw> ::bebop::SubRecord<'raw> for Library<'raw> {
 
 impl<'raw> ::bebop::Record<'raw> for Library<'raw> {}
 
+#[cfg(feature = "bebop-owned-all")]
+pub mod owned {
+    #![allow(warnings)]
+
+    use ::std::io::Write as _;
+    use ::core::convert::TryInto as _;
+    use ::bebop::FixedSized as _;
+
+    pub use super::Instrument;#[derive(Clone, Debug, PartialEq)]
```

Comment:
Might want to codegen a newline here. (Actually, I recall you setting up something to `rustfmt` the compiler output so that it's always pretty, or did I imagine that?)

Reply:
I thought I did fix that one already... Hm.
Yes, I do have rustfmt, but it is not enabled by default.